autoware_internal_planning_msgs/msg/CandidateTrajectories.msg

```diff
@@ -0,0 +1,2 @@
+autoware_internal_planning_msgs/CandidateTrajectory[] candidate_trajectories
+autoware_internal_planning_msgs/TrajectoryGeneratorInfo[] generator_info
```

Comment:
Could you explain which node would be filling in the generator_info fields? Is it going to be each trajectory generator? (if so, why do we want this to be an array?) 
Also, how do we ensure that each generator has a unique generator_id?